command-line-interface.md

```diff
@@ -0,0 +1,150 @@
+# Operations
+
+A conformant [runtime][] MUST provide an executable (called `funC` in the following examples).
+That executable MUST support commands with the following template:
+
+```sh
+$ funC [global-options] <COMMAND> [command-specific-options] <command-specific-arguments>
+```
+
+## Global options
+
+None are required, but the runtime MAY support options that start with at least one hyphen.
+Global options MAY take positional arguments (e.g. `--log-level debug`).
+Command names MUST NOT start with hyphens.
+The option parsing MUST be such that `funC <COMMAND>` is unambiguously an invocation of `<COMMAND>` (even for commands not specified in this document).
+If the runtime is invoked with an unrecognized command, it MUST exit with a nonzero exit code and MAY log a warning to stderr.
+
+## Character encodings
+
+This API specification does not cover character encodings, but runtimes SHOULD conform to their native operating system.
+For example, POSIX systems define [`LANG` and related environment variables][posix-lang] for [declaring][posix-locale-encoding] [locale-specific character encodings][posix-encoding], so a runtime in an `en_US.UTF-8` locale SHOULD write its [version](#version) to stdout in [UTF-8][].
+
+## Commands
+
+### version
```

Comment:
On Fri, Jul 08, 2016 at 09:50:59AM -0700, Mike Brown wrote:
> > +### version
>
> The runc implementation for outputting version information is `runc
> --version` or `runc -v`
> 
> I'm not a fan of the way it's implemented in runc, but there it is. 
> 
> Suggest moving the version command out for now.

Can't we just wrap runC in an argument-translator for testing?  “Print
your version and exit” sounds very command-like to me, and I'd rather
not overhaul the initial two sections here to say “and when you use a
command-like global option, you don't need a COMMAND”.